src/smt/theory_str.cpp

```diff
@@ -6284,6 +6295,10 @@ namespace smt {
             }
 
             TRACE("str", tout << "range NFA: start = " << start << ", end = " << end << std::endl;);
+        } else if (u.re.is_full(e)) {
+            NOT_IMPLEMENTED_YET();
```

Comment:
Oops, my mistake. I actually just caught this as well and came here to make sure this hadn't been merged yet.